docs/infra/quickstart.md

```diff
@@ -0,0 +1,86 @@
+---
+head:
+  - - meta
+    - name: "twitter:title"
+      content: Quickstart | zkSync Docs
+---
+
+# Quick Start Guide for zkSync Node
+
+## Prerequisites
+
+- **Installations Required:**
+  - [Docker Compose](https://docs.docker.com/compose/install/)
+  - [Docker](https://docs.docker.com/get-docker/)
+
+## Running a zkSync Node Locally
+
+### Starting the Node
+
+- **For a Mainnet instance:**
+
+  ```sh
+  cd docker-compose-examples
```

Comment:
This requires that users pull down the monorepo and cd into the specific directory, we should suggest users do the following at the top of the document:

```sh
git clone https://github.com/matter-labs/zksync-era.git
cd zksync-era/docs/guides/external-node
```

Reply:
Added